.github/workflows/benchmarks.yml

```diff
@@ -38,12 +44,14 @@ jobs:
       run:
         working-directory: ${{ env.WORKING_DIRECTORY }}
     permissions:
-      contents: read
+      contents: write
       id-token: write
+      pull-requests: write
```

Comment:
Yes, we need `pull-requests: write` since my current idea is to raise a PR for every benchmark PGO update so we can quickly eyeball it and confirm that the source benchmarks that created it was not a failure. After we get some confidence with this approach I'd move forward with auto merging PGO.

Reply:
See the current workflow action [here](https://github.com/elastic/apm-server/pull/13884/files#diff-22eef62c9d7b36d02f5fd5aaada92702e22d6795c464239cb7b8fe0f26ea1e1cR195) that requires the access.